mof_array/pmf/process_mass_data.py

```diff
@@ -56,80 +61,38 @@ def write_output_data(filename, data):
+# Keyword arguments:
+#   mofs_list -- list of MOF structures simulated
+#   experimental_mass_import -- dictionary formatted experimental results for each mof
+#   mof_densities -- dictionary of densities
+#   gases -- list of gases in simulated mixtures
+def import_experimental_results(mofs_list, exp_results_import, mof_densities, gases):
+    # Can probibily omit exp_results_mass in the future and just use the full
+    # dictionary for furthere analysis.
+    exp_results_full = []
+    exp_results_mass = []
+    exp_mofs_list = []
     for mof in mofs_list:
+        for row in exp_results_import:
+            if row['MOF'] == mof:
+                mass = float(mof_densities[mof]) * float(row['Mass'])
+                exp_results_full.extend([dict_copy_update(row,{'Mass_mg/cm3' : str(mass)})])
+                exp_results_mass.append({'MOF' : mof, 'Mass' : str(mass)})
+                exp_mofs_list.append(str(mof))
+            else:
+                None
+    return(exp_results_full, exp_results_mass, exp_mofs_list)
 
-        # Calculates masses in terms of mg/(cm3 of framework)
-        masses = [float(mof_densities[row['MOF']]) * float(row['Mass']) for row in
-                    experimental_mass_import if row['MOF'] == mof]
-        if len(masses) is not 0:
-
-            experimental_mofs.append(str(mof))
-            # Saves composition values as a list, necessary type for the Delaunay input argument
-            comps = []
-            for row in experimental_mass_import:
-                if row['MOF'] == mof:
-                    comps.extend([[float(row[gas]) for gas in gases]])
-
-            # List of experimental masses for current mof
-            experimental_mass_temp = [ row for row in experimental_mass_import if row['MOF'] == mof]
-
-            for index in range(len(masses)):
-                temp_dict = experimental_mass_temp[index].copy()
-                temp_dict.update({ 'Mass_mg/cm3' : masses[index] })
-                experimental_results.extend([temp_dict])
-
-            # Dictionary format of all the experimental data
-            temp_list = {'MOF' : mof, 'Mass' :[row['Mass_mg/cm3'] for row in experimental_results if row['MOF'] == mof]}
-            experimental_mass_mofs.append(temp_list)
-
-        else:
-            None
-
-    return(experimental_results, experimental_mass_mofs, experimental_mofs)
-
-def import_simulated_data(mofs_list, all_results, mof_densities, gases):
-    """Imports simulated data and puts it in dictionary format
-    If desired, interpolation is performed and may be used to create a denser data set
-
-    Keyword arguments:
-    mofs_list -- names of all mofs
-    all_results -- imported csv of outputs as dictionary
-    mof_densities -- dictionary of densities for each mof
-    gases -- list of all gases
-    """
-    simulated_results = []
+def import_simulated_data(mofs_list, sim_results_import, mof_densities, gases):
```

Comment:
Can we delete this entirely and just use the other function. so instead of:

full_data = import_simulated_data(...)

full_data, _, _ = import_experimental_results(...)